src/space.js

```diff
@@ -46,6 +47,10 @@ export default class ColorSpace {
 			this.formats.color.id = this.id;
 		}
 
+		if (options.idAliases) {
+			this.formats.color.ids = options.idAliases;
```

Comment:
I don't think we need this. They can just be defined in the format itself.

Reply:
Good point! Removed.